sys/vm/vm_page.h

```diff
@@ -444,6 +459,8 @@ extern struct mtx_padalign pa_lock[];
 #define	PGA_NOSYNC	0x0080		/* do not collect for syncer */
 #define	PGA_SWAP_FREE	0x0100		/* page with swap space was dirtied */
 #define	PGA_SWAP_SPACE	0x0200		/* page has allocated swap space */
+#define	PGA_CAPSTORE 	0x0400		/* Fast-path capdirty */
```

Comment:
Might be worth allocating 0x2000 and 0x4000 instead so that merge conflicts are more trivial to resolve (i.e. are just taking both sides' changes, rather than having to bump up these values each time).

Reply:
Not `0x8000` and `0x4000`?  Did one of our other branches take `0x8000`?

Reply:
I guess that's fine. I was thinking of 32-bit ints where you want to steer clear of 2^31 in constants (`prot` is `int` not `unsigned`), but this is `uint16_t` so we can use all bits without risk.